src/test/common/TestdataGenerator.kt

```diff
@@ -1194,3 +1205,128 @@ fun lagSanityBegrunnelse(
         skalAlltidVises = false,
         resultat = resultat,
     )
+
+fun lagSammensattKontrollsak(
+    id: Long = 0L,
+    behandlingId: Long,
+    fritekst: String = "",
+): SammensattKontrollsak =
+    SammensattKontrollsak(
+        id,
+        behandlingId,
+        fritekst,
+    )
+
+fun lagVedtakFellesfelterSammensattKontrollsakDto(
+    enhet: String = "enhet",
+    saksbehandler: String = "saksbehandler",
+    beslutter: String = "beslutter",
+    søkerNavn: String = "søkerNavn",
+    søkerFødselsnummer: String = "søkerFødselsnummer",
+    sammensattKontrollsakFritekst: String = "sammensattKontrollsakFritekst",
+) = VedtakFellesfelterSammensattKontrollsakDto(
+    "enhet",
+    "saksbehandler",
+    "beslutter",
+    "søkerNavn",
+    "søkerFødselsnummer",
+    "sammensattKontrollsakFritekst",
```

Comment:
Burde det ikke her være: 
```suggestion
    enhet,
    saksbehandler,
    beslutter,
    søkerNavn,
    søkerFødselsnummer,
    sammensattKontrollsakFritekst,
```

Slik at man har mulighet til å overstyre verdiene?

Reply:
Good catch!